benchmarks/util/run_and_upload.py

```diff
@@ -1,11 +1,13 @@
+from __future__ import print_function
+from __future__ import absolute_import
 import argparse
 import os
 import re
 import copy
 import uuid
 import calendar
 import time
-import big_query_utils
+from . import big_query_utils
```

Comment:
Why this change is related?

Reply:
Python 2 tolerates relative imports but Python 3 does not without the leading dot.